Makefile

```diff
@@ -7,17 +7,22 @@ all: lint test
 lint: pylint pep8
 test: test-unit test-integration test-functional
 
-init:
+.pre-init:
+	pip install --upgrade pip
+	@# Increase pylint version to remove restriction.
```

Comment:
I'm not sure what this comment means.

Reply:
Ok, I'll try to clarify it in the code. Basically the setuptools is breaking on a dependency of pylint that we can't upgrade until pylint is upgraded.